tests/foreman/api/test_config_template.py

```diff
@@ -30,56 +30,36 @@ def test_build_pxe_default(self):
         response.raise_for_status()
         self.assertIsInstance(response.json(), dict)
 
-    def test_associate_config_template_with_organization(self):
-        """@Test: Associate config template with organization
+    def test_config_template_orgs(self):
+        """@Test: Associate a config template with organizations.
 
         @Assert: Config template is associated with organization
 
         @Feature: ConfigTemplate
 
         """
-        org_1 = entities.Organization().create()
-        org_2 = entities.Organization().create()
+        orgs = [entities.Organization().create() for _ in range(2)]
 
         # By default, a configuration template should have no organizations.
         conf_templ = entities.ConfigTemplate().create()
         self.assertEqual(0, len(conf_templ.organization))
 
         # Associate our configuration template with one organization.
-        client.put(
-            conf_templ.path(),
-            {'config_template': {'organization_ids': [org_1.id]}},
-            verify=False,
-            auth=get_server_credentials(),
-        ).raise_for_status()
-
-        # Verify that the association succeeded.
-        conf_templ = conf_templ.read()
-        self.assertEqual(1, len(conf_templ.organization))
-        self.assertEqual(org_1.id, conf_templ.organization[0].id)
+        conf_templ.organization = orgs[:1]
+        conf_templ = conf_templ.update(['organization'])
```

Comment:
This line modifies just the local object:

```python
conf_templ.organization = orgs[:1]
```

This line sends an HTTP PUT request to the server, uses the response to construct a new `Organization` object, and save the new object as `conf_templ`:

```python
conf_templ = conf_templ.update(['organization'])
```

Why are we doing things this way? As an analogy, imagine that you are interacting with a website that translates text from english to german. This website has HTML, but no JavaScript. How do you send a request to the server? You do it like so:

1. Use a web browser to open the relevant page on this web page.
2. Add some english text in to a form field on that page.
3. Click "submit", capture the server's response, and use the response to build a new web page.

These three steps are roughly analagous to the following:

```python
conf_templ = ConfigTemplate(...)  # open web page
conf_templ.organization = orgs[:1]  # type in english text
conf_templ = conf_templ.update(['organization'])  # click "submit" and capture the response
```